src/components/LeftSidebar/LeftSidebar.vue

```diff
@@ -21,96 +21,77 @@
 
 <template>
 	<NcAppNavigation :aria-label="t('spreed', 'Conversation list')">
-		<div class="new-conversation"
-			:class="{ 'new-conversation--scrolled-down': !isScrolledToTop }">
-			<SearchBox v-model="searchText"
-				class="conversations-search"
-				:is-searching="isSearching"
-				@input="debounceFetchSearchResults"
-				@submit="onInputEnter"
-				@keydown.enter.native="handleEnter"
+		<div class="new-conversation" :class="{ 'new-conversation--scrolled-down': !isScrolledToTop }">
```

Comment:
So, something in Eslint configuration on your local PC is wrong, and a lot of lines were changed, which shouldn't happen usually.
Please, check for Eslint plugin settings, and don't forget to run `lint:fix` script

Reply:
Yes, I removed the default configuration so it won't cause that change again